odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1687,34 +1607,146 @@
+            <field name="active" eval="False"/>
+        </record>
+
+        <record id="UYW" model="res.currency">
+            <field name="name">UYW</field>
+            <field name="full_name">Unidad previsional</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.0001</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">peso</field>
+            <field name="currency_subunit_label">centésimo</field>
+        </record>
+
+        <record id="UYI" model="res.currency">
+            <field name="name">UYI</field>
+            <field name="full_name">Uruguay Peso en Unidades Indexadas</field>
+            <field name="symbol">$</field>
+            <field name="rounding">1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Peso</field>
+            <field name="currency_subunit_label">centésimo</field>
+        </record>
+
+        <record id="USN" model="res.currency">
+            <field name="name">USN</field>
+            <field name="full_name">United States dollar (next day)</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Dollars</field>
+            <field name="currency_subunit_label">Cents</field>
+        </record>
+
+        <record id="STN" model="res.currency">
+            <field name="name">STN</field>
+            <field name="full_name">São Tomé and Príncipe dobra</field>
+            <field name="symbol">Db</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Dobra</field>
+            <field name="currency_subunit_label">cêntimo</field>
+        </record>
+
+        <record id="MXV" model="res.currency">
+            <field name="name">MXV</field>
+            <field name="full_name">Mexican Unidad de Inversion</field>
+            <field name="symbol">$</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Pesos</field>
+            <field name="currency_subunit_label">Centavos</field>
+        </record>
+
+        <record id="CHW" model="res.currency">
+            <field name="name">CHW</field>
+            <field name="full_name">WIR franc (complementary currency)</field>
+            <field name="symbol">fr</field>
+            <field name="rounding">0.1</field>
+            <field name="active" eval="False"/>
+            <field name="currency_unit_label">Franc</field>
+            <field name="currency_subunit_label">Centimes</field>
+        </record>
+
+        <record id="CHE" model="res.currency">
+            <field name="name">CHE</field>
+            <field name="full_name">WIR euro (complementary currency)</field>
```

Comment:
```suggestion
            <field name="full_name">WIR euro</field>
```
@mart-e Same as above.

Reply:
Same, I could accept this is a localisation module if explicitly asked but I don't think we need this in base.

Reply:
@mart-e @AntoineVDV @mba-odoo Changes done